safeeyes/safeeyes/config/lang/ru.json

```diff
@@ -0,0 +1,43 @@
+{
+    "meta_info": {
+        "language_name": "Русский",
+        "language_name_en": "Russian"
+    },
+    "exercises": {
+        "long_break_exercises": [
+            "Пройдитесь немного",
+            "Откиньтесь на спинку стула и расслабьтесь"
+        ],
+        "short_break_exercises": [
+            "Сильно зажмурьтесь",

```

Comment:
Google Translate could not translate 'зажмурьтесь' meaningfully. Could you please verify this is correct or not?
I know Google Translate is not perfect but I use it to avoid human errors like typo. If you confirm that the above translation is correct, I will merge it.